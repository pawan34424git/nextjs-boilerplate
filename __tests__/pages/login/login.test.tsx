import LoginPage from "@/pages/login";
import "@testing-library/jest-dom";
import renderer from "react-test-renderer";

describe("login page", () => {
  it("login page rendering", () => {
    const tree = renderer.create(<LoginPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
