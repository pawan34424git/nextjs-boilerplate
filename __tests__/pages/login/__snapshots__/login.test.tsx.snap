// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login page login page rendering 1`] = `
<div
  className="min-h-full"
>
  <main>
    <div
      className="flex-col rounded m-0 p-0 dark:bg-slate-800  shadow-inner px-2 py-1 "
    >
      Login
    </div>
  </main>
</div>
`;
